packages/datagateway-common/src/table/columnFilters/dateColumnFilter.component.tsx

```diff
@@ -85,6 +93,42 @@ interface DateColumnFilterProps {
   filterByTime?: boolean;
 }
 
+const CustomTextField: React.FC<TextFieldProps> = (
+  renderProps: TextFieldProps & JSX.IntrinsicAttributes
+) => {
+  const invalidDateRange = renderProps.inputProps?.invalidDateRange;
+  const errorText = renderProps.inputProps?.errorText;
+  const filterByTime = renderProps.inputProps?.filterByTime;
+
+  // remove the custom props so they are not passed to DOM
+  delete renderProps.inputProps?.invalidDateRange;
+  delete renderProps.inputProps?.errorText;
+  delete renderProps.inputProps?.filterByTime;
```

Comment:
The same object destructuring comments I made in search date pickers also applies here. Just make sure to explicitly set `inputProps={inputProps}` in the `TextField`